ethereum_history_api/circuits/lib/src/fixtures/mainnet/cancun/transfer/transaction.nr

```diff
@@ -0,0 +1,21 @@
+use crate::transaction::TxPartial;
+
+global transaction = TxPartial {
+  nonce: 0,
+  gas_limit: 23100,
+  to: Option::some([
+    0xac, 0xd8, 0x8b, 0xe8, 0x98, 0xea, 0x10, 0xe6, 0x6c, 0x74, 0x42, 0xc4, 0xc2, 0x5d, 0x2c, 0x2d, 0x90, 0x7b, 0x5b, 0x0d
+  ]),
+  value: 1000000000000000,
+  data: [
+    
```

Comment:
Why is data empty when `data_len` is two?

Reply:
My mistake, changed data_len computation